scripts/release/test/util/smoke_test.sh

```diff
@@ -39,9 +36,9 @@ while [ "$1" != "" ]; do
     shift
 done
 
-if [ -z "$BRANCH" ] || [ -z "$HASH" ] || [ -z "$RELEASE" ]
+if [ -z "$BRANCH" ] || [ -z "$COMMIT_HASH" ] || [ -z "$FULLVERSION" ]
 then
-    echo "$YELLOW_FG[Usage]$END_FG_COLOR $0 -b BRANCH -c CHANNEL -h HASH -r RELEASE"
+    echo "[ERROR] $0 -b BRANCH -c CHANNEL -h COMMIT_HASH -r FULLVERSION"
```

Comment:
bug: The variables on this line need to be inside ${...} to be printed correctly.

Reply:
I don't want to print the value of the variables here, I want the error message to print something like:
```
[ERROR] ./foo.sh -b BRANCH -c CHANNEL -h COMMIT_HASH -r FULLVERSION
```

Reply:
How would that error message help you to diagnose that the wrong FULLVERSION was passed in ?

Reply:
I was originally thinking that it would act more like a "usage" message, but you're right that it would be better displaying the values.  I'll change it.